src/Plugin/jsonapi/FieldEnhancer/CardLinkEnhancer.php

```diff
@@ -132,6 +134,17 @@ public function getCardFields(EntityInterface $node) {
         $paragraph = $paragraph_id ? Paragraph::load($paragraph_id) : '';
         if ($paragraph && $paragraph->field_paragraph_date_range->getValue()) {
           $event_date = $paragraph->field_paragraph_date_range->getValue()[0];
+          // Parse date with GMT timezone.
+          $storageTz = DateTimeItemInterface::STORAGE_TIMEZONE;
+          $dateValue = new DrupalDateTime($event_date['value'], $storageTz);
```

Comment:
Please verify value and end_value exist.

Reply:
Will verify the date_event as date event will either return the date range value or NULL.